website/content/en/status/report-2023-07-2023-09/mfsbsd.adoc

```diff
@@ -12,19 +12,22 @@ Contact: Soobin Rho <soobinrho@FreeBSD.org>
 "mfsBSD is a toolset to create small-sized but full-featured mfsroot based distributions of FreeBSD that store all files in memory (MFS) [Memory File System] and load from hard drive, USB storage device, or optical medium.
 It can be used for a variety of purposes, including diskless systems, recovery partitions and remotely overwriting other operating systems."
 
-mailto:mm@FreeBSD.org[Martin Matuska] is the creator of mfsBSD, and he is also the author of the original link:https://people.freebsd.org/~mm/mfsbsd/mfsbsd.pdf[mfsBSD white paper], which is where the excerpt above was taken from.
-The upstream mfsBSD is maintained at the link:https://github.com/mmatuska/mfsbsd[GitHub mfsBSD repository].
+http://blog.vx.sk/[Martin Matuška] is the creator of mfsBSD.
+He is also author of the original (2009) link:https://people.freebsd.org/~mm/mfsbsd/mfsbsd.pdf[mfsBSD white paper], from which the excerpt above is taken.
+Upstream mfsBSD is maintained in the link:https://github.com/mmatuska/mfsbsd[repository on GitHub].
 
 ==== Purpose of this Project
 
-This project integrates mfsBSD into the FreeBSD release tool set, creating an additional target of mfsBSD images ([.filename]#.img# and [.filename]#.iso# files) into the [.filename]#/usr/src/release/Makefile#.
-Until now, mfsBSD only existed outside the FreeBSD release tool chain, and only the -release versions have been produced.
-With this project, mfsBSD images will be available at the official FreeBSD release page, and you will also be able to build mfsBSD yourself by invoking `cd /usr/src/release && make release WITH_MFSBSD=1`, which will then create [.filename]#mfsbsd-se.img# and [.filename]#mfsbsd-se.iso# at [.filename]#/usr/obj/usr/src/${ARCH}/release/#.
+This project integrates mfsBSD into the FreeBSD release tool set, creating an additional target of mfsBSD images ([.filename]#.img# and [.filename]#.iso# files) in [.filename]#/usr/src/release/Makefile#.
+Prior to integration, mfsBSD only existed outside the FreeBSD release tool chain, and only -RELEASE versions were produced.
```

Comment:
I changed _-release_ to uppercase _-RELEASE_. 

Is this appropriate?

Reply:
That will be a good change!